sass/utils/_functions.scss

```diff
@@ -0,0 +1,19 @@
+// ACCESS COLORS FUNCTION
+
+@function color($key: 'primary') {
+  @return map-get($theme-colors, $key);
+}
```

Comment:
I am very impressed with your use of the list-map structures and functions. This is an interesting approach to handling colors.